### PR TITLE
Make tab navigation not use hrefs but rather $location

### PR DESCRIPTION
**Pull request by akariv**
Fixes https://github.com/openspending/openspending/issues/860

### CODE REVIEW

File: app/views/partials/steps.html [hunk 1]
```diff
@@ -2,10 +2,11 @@
 <ul class="nav nav-pills" id="steps-nav">
   <li role="presentation" ng-repeat="step in steps"
     ng-class="{active: step.id == currentStep.id, disabled: !step.isCurrent && !step.isPassed}">
-    <a ng-href="{{ step.isPassed || step.isCurrent ? step.route : 'javascript:void(0)' }}">
+    <a href='#'
+       ng-click="goToStep(step)">
```
Comment (brew) [inline app/views/partials/steps.html:6]: Is this line break an angular convention?

**Comment by brew** [comment]
LGTM, with a small comment.

File: app/views/partials/steps.html (hunk 1 above)
Comment (pwalsh) [inline app/views/partials/steps.html:6]: It is fairly common to see breaks like this in framework code (Angular, React) for readability, line length, etc.

**Comment by pwalsh** [comment]
@akariv can you remove shippable config pls

**Comment by akariv** [comment]
It seems like I don't have the creds to turn it off from shippable's side.

**Comment by pwalsh** [comment]
@akariv i've turned off shippable.

**Pull request merged**